internal/common/common.go

```diff
@@ -25,11 +25,25 @@ import (
 var (
 	// OperatorNamespace of the current operator instance, set in init()
 	OperatorNamespace                   string
-	InvalidObjectKeyError               = fmt.Errorf("invalid objectKey")
-	InvalidObjectKeyErrorEmptyName      = fmt.Errorf("%w, empty name", InvalidObjectKeyError)
-	InvalidObjectKeyErrorEmptyNamespace = fmt.Errorf("%w, empty namespace", InvalidObjectKeyError)
+	InvalidObjectKeyError                      = fmt.Errorf("invalid objectKey")
+	InvalidObjectKeyErrorEmptyName             = fmt.Errorf("%w, empty name", InvalidObjectKeyError)
+	InvalidObjectKeyErrorEmptyNamespace        = fmt.Errorf("%w, empty namespace", InvalidObjectKeyError)
+	defaultMaxRetries                   uint64 = 60
```

Comment:
nit: Is it me, or does something about the alignment feel off? I guess everything is aligned on `=`, but what about `OperatorNamespace`? Anyways, such a minor thing anyway.

Reply:
Yeah, I think that's  just the way that `go fmt` does it?